crates/mdman/tests/compare/expected/formatting.1

```diff
@@ -66,14 +66,8 @@ With a second paragraph inside it
 .sp
 .RS 4
 \h'-04'\(bu\h'+02'Milk
-.sp
-.RS 4
-\h'-04' 5.\h'+01'Don't start at one.
-.RE
-.sp
-.RS 4
-\h'-04' 6.\h'+01'tamarind
-.RE
+5. Don\[cq]t start at one.
+6. tamarind
```

Comment:
This looks incorrect. So does `formatting.txt`. I wonder there is some newline parsing change between versions of pulldown-cmark. Unfortunately, I don't have the bandwidth looking into this specific issue. If we cannot get through it, I might prefer to [the second option here](https://github.com/rust-lang/cargo/pull/11646#discussion_r1101478103), which is

> Don't use smart punctuations for pages that are shared between manpages and The Cargo Book.

Reply:
Just looked into it a bit. The new behavior completely conform to commonmark spec v0.30. See the explanation of [example 304](https://spec.commonmark.org/0.30/#example-304). In short, only with an extra newline before it, the first nested list item starting from non-zero can be seen as list item.

That is to say, the change here is correct. No need to change. See this [online demo] as well. Thank you for catching this subtle difference :)


[online demo]: https://spec.commonmark.org/dingus/?text=%23%23%20Lists%0A%0A1.%20Ordered%20list%0A%0A%20%20%20%20*%20Unordered%20list%0A%0A%20%20%20%20%20%20With%20a%20second%20paragraph%20inside%20it%0A%0A%20%20%20%20%20%20%20%201.%20Inner%20ordered%20list%0A%0A%20%20%20%20%20%20%20%201.%20Another%0A%0A%20%20%20%20*%20Eggs%0A%0A%20%20%20%20*%20Milk%0A%20%20%20%20%20%20%20%205.%20Don%27t%20start%20at%20one.%0A%20%20%20%20%20%20%20%206.%20tamarind%0A%0A1.%20Second%20element%0A%0A1.%20Third%20element%0A

Reply:
Can you please add a blank line to `crates/mdman/tests/compare/formatting.md` to fix this?